CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 ## Unreleased ([details][unreleased changes details])
 <!-- Keep this up to date! After a release, change the tag name to the latest release -->
 [unreleased changes details]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-4.3.2...HEAD
+- #2199 Add read permission for acs-commons-email-service user in conf folder using rep policy
```

Comment:
Please add a `### Fixed` header above this, to match the format of the rest of the file.

Reply:
Ill resolve this on merge - thanks @rajeevyadav2 and @HitmanInWis !